test/unit/math/mix/scal/fun/F32_test.cpp

```diff
@@ -1,4 +1,7 @@
 #include <stan/math/mix/scal.hpp>
+#include <stan/math/mix/mat.hpp>
```

Comment:
The test also includes `test/unit/math/rev/mat/fun/util.hpp`, which includes `stan/math/rev/mat.hpp` and thus `stan/math/rev/mat/vectorize/apply_scalar_unary.hpp`. So, it needs the `fwd` version of `stan/math/rev/mat/vectorize/apply_scalar_unary.hpp` to not error out. I thought the consistent way to do so was to add `stan/math/mix/mat.hpp`. Should I add `stan/math/fwd/mat.hpp` instead?